### DISCUSSION TITLE
Hi i cant make it work?

### DISCUSSION THREAD

**Discussion by GaldosV** [proyecto26/RestClient#217] [Q&A]
Hi i followed the steps to install but when i call it in visual studio with RestClient.Post("",User);it says that post does not exists in the namespaceRestClient i am doing the Use proyecto26 on the top . so why does it not work ?.

**Comment by maifeeulasad** [comment]
Hi @GaldosV, welcome.

Can you please create a project and share it with us. Maybe we can help. Else you can try sharing some related logs. And your environment details would also be helpful. Thanks.

- [ ] project list/logs and env details
